packages/@webex/widget-provider/package.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
should we rename this to something else , provider is too generic , should it be called widgets store or something ?

Reply:
`@webex/widgets-store` sounds good to me

Reply:
Changed to widgets-store

Reply:
This is now changed to `@webex/cc-store` as a final call